namespace-specific-policy/manual-rendering/README.md

```diff
@@ -17,25 +17,31 @@ This section describes prerequisites you must meet before this tutorial.
 ## Create namespace specific policies
 
 ### Get the example configuration
-This `manual-rendering` example contains three namespaces for different teams. It contains the following directories and files.
+This `manual-rendering` example contains four namespaces for different teams. It contains the following directories and files.
 ```
 .
 ├── configsync
+│   ├── external-team
+│   │   ├── external-team_networking.k8s.io_v1_networkpolicy_deny-all.yaml
```

Comment:
The folder structure needs to be updated.

Reply:
Done